examples/readthedocs-extend/docs/environment.yml

```diff
@@ -0,0 +1,6 @@
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  - pixi ==0.22.0
```

Comment:
Would be great to wait for a fix for this: https://github.com/conda-forge/pixi-feedstock/issues/46. With new build to be able to bump this to the latest version.

Reply:
Sure. No hurry.